recipe/meta.yaml

```
@@ -1,99 +1,95 @@
-        - pkg-config                         # [not win]
-        - make                               # [not win]
-      host:
-        - {{ pin_subpackage('xcb-proto') }}
-        - pthread-stubs
-    about:
-      summary: "This is the C-language Binding (XCB) package to the X Window System protocol"
+    - m2-autoconf                   # [win]
+    - m2-automake{{ am_version }}   # [win]
+    - m2-libtool                    # [win]
+    - pkg-config                    # [unix]
+    - m2-pkg-config                 # [win]
+    - gnuconfig                     # [unix]
+    - m2-base                       # [win]
+    - make                          # [unix]
+    - m2-make                       # [win]
+    - {{ compiler('c') }}           # [unix]
+    - {{ compiler('m2w64_c') }}     # [win]
+    - autoconf                      # [unix]
+    - automake                      # [unix]
+    - gettext                       # [unix]
+    - libtool                       # [unix]
+    - python  # [build_platform != target_platform]
+  host:
+    - pthread-stubs
+    - xorg-libxau
+    - xorg-libxdmcp
+    - xorg-util-macros
+    - xcb-proto {{ version }}
+    - xorg-xorgproto
+  run:
+    - pthread-stubs
+    - xorg-libxau
+    - xorg-libxdmcp
 
-  - name: xcb-proto
-    script: install_proto.sh
-    requirements:
-      build:
-        - {{ compiler('c') }}                # [not win]
-        - {{ cdt('xorg-x11-util-macros') }}  # [linux]
-        - m2-autoconf                        # [win]
-        - m2-automake{{ am_version }}        # [win]
-        - m2-libtool                         # [win]
-        - m2w64-pkg-config                   # [win]
-        - m2w64-toolchain                    # [win]
-        - posix                              # [win]
-        - pkg-config                         # [not win]
-        - make                               # [not win]
-        - python
-      host:
-        - python
-      run:
-        - python
-    test:
-      imports:
-        - xcbgen
-    about:
-      summary: "Provides the XML-XCB protocol descriptions that libxcb uses to generate the majority of its code and API"
+test:
+  commands:
+    {% set lib_idents = [
+      "xcb",
+      "xcb-composite",
+      "xcb-damage",
+      "xcb-dpms",
+      "xcb-dri2",
+      "xcb-glx",
+      "xcb-present",
+      "xcb-randr",
+      "xcb-record",
+      "xcb-res",
+      "xcb-screensaver",
+      "xcb-shape",
+      "xcb-shm",
+      "xcb-sync",
+      "xcb-xf86dri",
+      "xcb-xfixes",
+      "xcb-xinerama",
+      "xcb-xkb",
+      "xcb-xtest",
+      "xcb-xv",
+      "xcb-xvmc",
+    ] %}
+    {% set lib_idents = lib_idents + ['xcb-dri3', 'xcb-render', 'xcb-xinput'] %}  # [not win]
+    {% for lib_ident in lib_idents %}
+    - test -f $PREFIX/lib/lib{{ lib_ident }}.dylib  # [osx]
+    - test -f $PREFIX/lib/lib{{ lib_ident }}.so  # [linux]
+    {% endfor %}
 
 about:
-  home: https://xcb.freedesktop.org/
+  home: http://xcb.freedesktop.org/
```

```suggestion
  home: https://xcb.freedesktop.org/
```
Seems to be a valid URL to me.

---

Done